app/controllers/facilities_controller.rb

```diff
@@ -25,24 +26,25 @@ def new
   # POST /facilities
   def create
     facility = Facility.create(facilities_params)
-    user_saved = if !current_user.superuser?
+    user_saved =
+      if !current_user.superuser?
         user = User.add_to_facility(current_user.id, facility.id)
         user.save
       else
         true
       end
```

Comment:
I have doubt here @ProCode2 . Can you check it again?

Also we can create `add_to_facility` as it will be called as `current_user.add_to_facility(facility_id)`

Reply:
@abhinandanarya06 I am not auto assigning a facility to a user when they create it if the user is a superuser.

Yes, will do it the other way, while refactoring.
(This is unrelated to this PR)